test/Table_Tests/src/Data_Formatter_Spec.enso

```diff
@@ -1,52 +1,32 @@
 from Standard.Base import all
 
-import Standard.Table
-from Standard.Table import Column, Data_Formatter_Data, Quote_Style
+from Standard.Table import Table, Column, Data_Formatter, Quote_Style
 from Standard.Table.Errors import all
 
 import Standard.Test
 import Standard.Test.Problems
 
-# TODO Dubious constructor export
-from project.Data_Formatter_Spec.Custom_Type import all
-from project.Data_Formatter_Spec.Custom_Type export all
-
 type Custom_Type
-    Custom_Type_Data field
-
-# TODO Dubious constructor export
-from project.Data_Formatter_Spec.Custom_Type_With_To_Text import all
-from project.Data_Formatter_Spec.Custom_Type_With_To_Text export all
+    Value field
 
 type Custom_Type_With_To_Text
-    Custom_Type_With_To_Text_Data field
+    Value field
 
     to_text : Text
     to_text self = "[CUSTOM = " + self.field.to_text + "]"
 
-# TODO Dubious constructor export
-from project.Data_Formatter_Spec.Custom_Type_With_Error import all
-from project.Data_Formatter_Spec.Custom_Type_With_Error export all
-
 type Custom_Type_With_Error
-
     to_text : Text
     to_text self = Error.throw (Illegal_State_Error_Data "foo_error")
 
-# TODO Dubious constructor export
-from project.Data_Formatter_Spec.Custom_Type_With_Panic import all
-from project.Data_Formatter_Spec.Custom_Type_With_Panic export all
-
 type Custom_Type_With_Panic
-    Custom_Type_With_Panic_Data
-
     to_text : Text
     to_text self = Panic.throw (Illegal_State_Error_Data "foo_panic")
 
 spec =
     Test.group "DataFormatter.parse" <|
         Test.specify "should parse numbers" <|
-            formatter = Data_Formatter_Data
+            formatter = Data_Formatter.Value
```

Comment:
Is this the recommended way to create a new `Data_Formatter`?

It is a public type and I thought that we wanted `.Value` to be used only for internal stuff?

Maybe we should introduce a `Data_Formatter.new` method which delegates to `.Value` hiding it from the user?

Reply:
Still have the type rename to do - will do that in a separate PR.

Reply:
Ah,ok - sure! I just saw a few types getting renamed (like `Test_Selection.Config`) and thus thought about it. But definitely good to do this step by step.